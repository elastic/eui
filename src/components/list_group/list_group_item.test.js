import React from 'react';
import { render } from 'enzyme';

import {
  EuiListGroupItem,
} from './list_group_item';

describe('EuiListGroupItem', () => {
  test('is rendered', () => {
    const component = render(
      <EuiListGroupItem label="Label" />
    );

    expect(component)
      .toMatchSnapshot();
  });

  test('renders href', () => {
    const component = render(
      <EuiListGroupItem label="Label" href="#" />
    );

    expect(component)
      .toMatchSnapshot();
  });
});
