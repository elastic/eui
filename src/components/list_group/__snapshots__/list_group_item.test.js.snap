// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EuiListGroupItem is rendered 1`] = `
<li
  class="euiListGroupItem euiListGroupItem--medium"
>
  <span
    class="euiListGroupItem__text"
  >
    <span
      class="euiListGroupItem__label"
      title="Label"
    >
      Label
    </span>
  </span>
</li>
`;

exports[`EuiListGroupItem renders href 1`] = `
<li
  class="euiListGroupItem euiListGroupItem--medium euiListGroupItem-isClickable"
>
  <a
    class="euiListGroupItem__button"
    href="#"
  >
    <span
      class="euiListGroupItem__label"
      title="Label"
    >
      Label
    </span>
  </a>
</li>
`;
